src/Form/Helper/FormHelper.php

```diff
@@ -116,8 +121,37 @@ public static function resolveFormOptions($schema): array
                     default:
                         return $options;
                 }
+            case 'number':
+                switch (true) {
+                    case static::isRangeType($property):
+                        return $options + [
+                                'attr' => [
+                                    'min' => $property->minimum ?? $property->exclusiveMinimum,
```

Comment:
Shouldn't it be `$property->minimum ?? ($property->exclusiveMinimum +1)`

Reply:
The problem is that the attribute exclusiveMinimum / -Maximum has another logic in draft-04 as in draft-07. In draft-04 the attribute accept a boolean value while in draft-07 it accepts a number value. Sure it is possible to create another checkMethod to identify whether the value of the attribute is a boolean or a number, but I think this should be make in another PR. :-)

Reply:
Ok